src/main/java/bio/terra/app/controller/ApiValidationExceptionHandler.java

```diff
@@ -1,8 +1,11 @@
 package bio.terra.app.controller;
 
 
+import bio.terra.common.exception.DataRepoException;
 import bio.terra.model.ErrorModel;
+import org.apache.commons.lang.exception.ExceptionUtils;
```

Comment:
Probably want to use `lang3` here
```suggestion
import org.apache.commons.lang3.exception.ExceptionUtils;
```